package.json

```diff
@@ -21,7 +21,7 @@
     "cy:run:chrome": "yarn run cy:run --browser chrome",
     "cy:run:edge": "yarn run cy:run --browser edge",
     "cy:run:firefox": "yarn run cy:run --browser firefox",
-    "cy:run": "start-server-and-test 'yarn website:test' http://localhost:3000 'cypress run --spec 'cypress/integration/**/*' --headless'",
+    "cy:run": "start-server-and-test 'yarn playground:start' http://localhost:3000 'cypress run --spec 'cypress/integration/**/*' --headless'",
```

Comment:
I'm testing against the React version of demo, not the js version due to some problems I ran into when using the latter version like `yarn watch` won't output umd version. Anyway, could be changed later once those problems get fixed.

Reply:
No problem it's more important to test the react version since it's where all the logic belong